get all transaction by walletId

nuto17 opened the pull request.

nuto17 committed:
logic

nuto17 committed:
move transaction to walletService

yegorowns commented on src/main/java/com/example/financetracker/controller/WalletController.java:
```diff
@@ -60,5 +60,10 @@ public WalletDto operationWallet(@RequestBody TransactionDto transactionDto, @Pa
     }
+
+    @GetMapping("/{id}/transactions")
+    public List<TransactionDto> getAllTransactionsByType(@PathVariable Long id, @RequestParam TransactionType type) {
```
нужно сделать type необязательным, и если его нет то возвращать все транзакции

nuto17 replied:
done

yegorowns replied:
approved

yegorowns commented on src/main/java/com/example/financetracker/service/WalletService.java:
```diff
@@ -48,22 +51,23 @@ public void deleteWalletById(Long id) {
     }
 
     public void compareAmountBalance(Wallet wallet, BigDecimal amount) {
```
нужно изменить название метода на что то по типу "validateSufficientBalance"

nuto17 replied:
done

yegorowns replied:
approved

nuto17 committed:
1.rename method
2. move logic from TransactionService to WalletService
3.add required in request param

nuto17 committed:
a little fix
